frame/staking/src/benchmarking.rs

```diff
@@ -620,7 +620,11 @@ benchmarks! {
 		let (_, controller) = create_stash_controller::<T>(USER_SEED, 100, Default::default())?;
 		Staking::<T>::validate(RawOrigin::Signed(controller.clone()).into(), ValidatorPrefs::default())?;
 		Staking::<T>::update_staking_limits(
-			RawOrigin::Root.into(), BalanceOf::<T>::max_value(), BalanceOf::<T>::max_value(), None, None,
+			RawOrigin::Root.into(),
+			BalanceOf::<T>::max_value(),
+			BalanceOf::<T>::max_value(),
+			Some(0),
+			Some(0),
```

Comment:
This is because `Some(0)` forces a write, and `None` doesn't?

Reply:
No. 0 allows us to call chill other when none does not